### DISCUSSION TITLE
How to use login before join?

### DISCUSSION THREAD

**Discussion by Namnamphim** [Pryaxis/TShock#2985] [Q&A]
I recently ran a tshock server and I saw in the config.json file that there was an option. DisableLoginBeforeJoin. So I set it to false but nothing happened. Why isn't there a prompt to enter a password before entering the server?

**Comment by ghost** [comment]
I dont know...

**Comment by ghost** [comment]
New awnser: you must have login with UUID enabled and a server password set (set the server password when starting the server) to use login before join... otherwise you would have to enter the server password if `"UUID"=false`, `"Login before join"=false` or there is no server password.
Does that help???
